src/app/public/modules/fluid-grid/fluid-grid.component.ts

```diff
@@ -39,7 +39,7 @@ export class SkyFluidGridComponent {
   }
 
   public get gutterSize(): SkyFluidGridGutterSize {
-    return this._gutterSize || SkyFluidGridGutterSize.Large;
```

Comment:
If the consumer initially sets `gutterSize` to `SkyFluidGridGutterSize.Small` this line was evaluating that enum as `0`, and evaluating as `false`, thus reverting to its default of `Large`.